Uncap oxygenation for asystole mobs

Qlonever opened the pull request:
<!-- 
Do not forget to add a changelog when you have made admin/player facing changes that can alter gameplay.
Examples which require a changelog entry include:
* Adding/removing objects that players may interact with, or the way they function.
* Adding/removing/altering admin tools.
* Changing the map.

Examples were changelog entries are optional/not typically required:
* Cosmetic changes such as descriptions, sound effects, etc.
* Optimizations and other changes to underlying systems which do not affect gameplay.
* Minor bug fixes.

You'll find a README and example file in .\html\changelogs\ for further instructions.

You can also find a template for adding your changelog directly to the PR description here: https://github.com/Baystation12/Baystation12/wiki/Automatic-changelog-generation
-->
:cl:
tweak: Increased the effectiveness of chest compressions. Manual compressions can actually be more effective than automatic ones now.
/:cl:

Removes the 30% oxygenation cap for asystole mobs.

Reasoning:
- Asystole is already considered by `get_blood_circulation()`, and effectively caps oxygenation at 25% without CPR. It shouldn't need to be capped again.

- The wiki states that manual CPR is more effective than compression from auto-compressors, and their code indicates that this was clearly the intent. However, capping oxygenation at 30% makes this difference completely useless.

- The removed conditional caused `get_blood_oxygenation()` to return before taking respiration/dexalin into account, which is probably undesirable.

If necessary I can adjust the circulation granted by CPR and autocompressors, since masterful compressions grant 80-100% circulation, and automatic compressions grant 60% circulation. (Ignoring other factors like blood loss and respiration) Those values seem like they might be too high.

Shouldn't be merged without #34670 or else manual compressions will grant more than 100% circulation half of the time.

Qlonever committed:
uncap circulation from cpr

Spookerton commented:
Could stand to be more detailed since is_asystole covers additional things like entirely missing hearts, but fine as an intermediate step.

Spookerton commented:
It's worth keeping in mind that manual CPR gets adjusted by skill and is hugely faster than autos, which run every 15 seconds, where manual CPR ranges from 2.1 to 0.9 seconds. I think the rate contributes a lot already to the effective effectiveness 🙃 of manual.

In terms of effectiveness autocomps are fixed 0.6 and manual ranges from worst 0.3 to best 1.0. Since we certainly don't *want* 1.0, changing `0.1*pumping_skill` to `0.05*` gives us 0.3 to 0.65, which seems plenty high to me. Autocomps can go down to 0.5 and run every 10 seconds in trade, just to see how that plays out.

Spookerton committed:
adjust auto and manual cpr effectiveness

auto: 0.6 every 15 => 0.5 every 10
manual: 0.3~1.0 every 2.1~0.9 => 0.3~0.65 every 2.1~0.9

Qlonever commented:
I'm not sure if you'll see this, but:
>It's worth keeping in mind that manual CPR gets adjusted by skill and is hugely faster than autos, which run every 15 seconds, where manual CPR ranges from 2.1 to 0.9 seconds. I think the rate contributes a lot already to the effective effectiveness 🙃 of manual.

`get_blood_circulation()` only checks for compressions in the last 20 seconds, so any compressions faster than that are equally effective as each other in terms of speed.

> As for the actual change itself, I'm inclined to say CPR should still be ineffective if the recipient literally lacks a heart, since is_asystole() did cover that as well. I'll do that later though.

`get_blood_circulation()` checks for a heart before anything else, and returns 0.25 * blood volume without factoring in compressions if there isn't one.
